### PR TITLE
PRM benchmarking

### PR DESCRIPTION
**Pull request by ilansmith**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/936/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/782/ for details (Mellanox internal link).

File: src/vma/sock/sock-redirect.cpp [hunk 1]
```diff
@@ -375,19 +375,19 @@ int vma_poll(int fd, struct vma_completion_t* completions, unsigned int ncomplet
 
 		ret_val = p_ring->vma_poll(completions, ncompletions, flags);
 #ifdef RDTSC_MEASURE_RX_PROCCESS_BUFFER_TO_RECIVEFROM
-	RDTSC_TAKE_END(g_rdtsc_instr_info_arr[RDTSC_FLOW_PROCCESS_RX_BUFFER_TO_RECIVEFROM]);
+	RDTSC_TAKE_END(RDTSC_FLOW_PROCCESS_RX_BUFFER_TO_RECIVEFROM);
```
Comment (OphirMunk) [inline src/vma/sock/sock-redirect.cpp]: Here the measurement ends:
```
RDTSC_TAKE_END(RDTSC_FLOW_PROCCESS_RX_BUFFER_TO_RECIVEFROM);
```
Where does is start? Where is
```
RDTSC_TAKE_START(RDTSC_FLOW_PROCCESS_RX_BUFFER_TO_RECIVEFROM);
```
???

File: src/vma/sock/sock-redirect.cpp [hunk 2]
```diff
@@ -375,19 +375,19 @@ int vma_poll(int fd, struct vma_completion_t* completions, unsigned int ncomplet
...
 #endif //RDTSC_MEASURE_RX_PROCCESS_BUFFER_TO_RECIVEFROM
 
 #ifdef RDTSC_MEASURE_RX_LWIP_TO_RECEVEFROM
-	RDTSC_TAKE_END(g_rdtsc_instr_info_arr[RDTSC_FLOW_RX_LWIP_TO_RECEVEFROM]);
+	RDTSC_TAKE_END(RDTSC_FLOW_RX_LWIP_TO_RECEVEFROM);
```
Comment (OphirMunk) [inline src/vma/sock/sock-redirect.cpp]: Here the measurement ends:
```
RDTSC_TAKE_END(RDTSC_FLOW_RX_LWIP_TO_RECEVEFROM);
```
Where does is start? Where is
```
RDTSC_TAKE_START(RDTSC_FLOW_RX_LWIP_TO_RECEVEFROM);
```
???

**Comment by ilansmith** [comment]
bot:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1010/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/accl-libvma-pr/804/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/accl-libvma-pr/1033/ for details (Mellanox internal link).

**Comment by ilansmith** [comment]
Replaced by #400.

**Pull request closed without merging**